docs/UserGuide.md

```diff
@@ -279,7 +279,35 @@ Clears all entries from TR4CKER.
 
 Format: `clear`
 
-#### Daily tab features
+#### Daily tab features (Yingqi)
+You can add all your daily plans of current tasks to a daily to do list so that you have a clearer idea of what you want to complete for the day.
+
+##### Switching to Daily tab: `daily`
+If you would like to switch to Daily tab without clicking on the Daily tab button, you can do so easily through the CLI, by just typing `daily`.
+
+Format: `daily`
+
+Once you enter `daily` into the command box, TR4CKER should look like this:
+![switch tab](images/dailyTab_switchTab.png)
+Figure 1: TR4CKER after executing `daily`.
+
+Example: 
+* `daily` - Switches to Daily tab, showing a list of daily todo tasks that you have planned for the day.
+
+##### Adding a daily todo task: `todo`
+
+You can add a todo task for the day into the daily todo list by providing the index number of the task that you want to add.
+
+Format: `todo INDEX` [To be implemented: add multiple todo tasks at the same time]
+
+* Adds the task at the specified `INDEX` to daily todo list.
+* The index refers to a valid index number shown in the displayed task list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `todo 1` adds the first task in the displayed task list into daily todo list.
+![todo result](images/dailyTab_todo1.png)
```

Comment:
It might be helpful to show a before and after of executing the command, since the adding of the todo can be seen in a different tab.